docs/src/pages/[platform]/getting-started/migration/migration.vue.mdx

```diff
@@ -1,25 +1,93 @@
 import { Tabs, TabItem } from '@aws-amplify/ui-react';
 import { Alert } from '@aws-amplify/ui-react';
 
+## 2.x to 3.x
+### Installation
+For Vue.js 2 users please continue to use our legacy libary found [here](https://github.com/aws-amplify/amplify-ui/tree/legacy/legacy/amplify-ui-vue).
+
+Install the 3.x version of the `@aws-amplify/ui-vue` library.
+
+<Tabs>
+<TabItem title="npm">
+
+```shell
+npm install aws-amplify @aws-amplify/ui-vue@3.x
+```
+
+</TabItem>
+<TabItem title="yarn">
+```shell
+yarn add aws-amplify @aws-amplify/ui-vue@3.x
+```
+
+</TabItem>
+</Tabs>
+
+### Update and usage
+
+The following breaking changes were introduced in `@aws-amplify/ui-vue` major version `3.0`:
+
+#### 1. Update any override function calls
+
+Update the override function calls with the updated function signatures. Rather then returning an `Auth.*` object you must now return the function callback passed in the `handle*` signature.
```

Comment:
```suggestion
Update the function override calls with the updated function signatures. Rather than returning an `Auth.*` object you must now return the function callback passed in the `handle*` signature.
```